openlibrary/catalog/merge/merge_bot/bot.py

```diff
@@ -1,21 +1,19 @@
 #!/usr/bin/python
 
 from __future__ import print_function
+
+import codecs
 import sys
+
 import web
-import sys
-import codecs
-from catalog.utils.query import query_iter, set_staging, withKey
 
-from catalog.merge.merge_marc import *
```

Comment:
how can we be sure we're getting everything here

Reply:
https://www.asmeurer.com/removestar